packages/expo-env-info/jest.config.js

```diff
@@ -1,7 +1,11 @@
-const path = require('path');
+import { dirname, resolve } from 'path';
+import { fileURLToPath } from 'url';
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
 
-module.exports = {
+export default {
   preset: '../../jest/unit-test-config',
-  rootDir: path.resolve(__dirname),
-  displayName: require('./package').name,
+  rootDir: resolve(__dirname),
+  displayName: 'expo-env-info',
```

Comment:
Doesn't `require` work here? Maybe we should use `fs` instead?

Reply:
Yes, I initially changed this to use `fs`, but as this is not a generic config, but one specific to the project, there's really no point in the overhead of reading the package.json for something we already know (imho).